kafka-ui-react-app/src/components/common/PropertiesList/PropertiesList.styled.tsx

```diff
@@ -12,11 +12,11 @@ export const List = styled.div`
 export const Label = styled.div`
   font-size: 14px;
   font-weight: 500;
-  color: ${({ theme }) => theme.list.label.color};
+  color: ${({ theme }) => theme.default.color.normal};
   white-space: nowrap;
 `;
 
-export const SubText = styled.div`
-  color: ${({ theme }) => theme.list.meta.color};
-  font-size: 12px;
-`;
+// export const SubText = styled.div`
```

Comment:
if we don't need the component lets delete it.

Reply:
done